bootstrap

```diff
@@ -34,23 +34,23 @@ squawk () {
       printf "#"
       count_squawk=`expr $count_squawk + 1`
     done
-    echo " $squawk"
+    printf " $squawk \n"
   fi
 }
 
 # Check for config env file
+# THIS IS NOT USED, AND SHOULD BE REMOVED
 if [ -e $TRANSMUTE_DIR/.env ]; then
   source $TRANSMUTE_DIR/.env
   if [ $TRANSMUTE_VERBOSITY -gt '0' ]; then
-    ls -al $TRANSMUTE_DIR/.env
-    env file found and sourced
+    # ls -al $TRANSMUTE_DIR/.env
+    printf "env file found and sourced.\n"
   fi
 fi
 
 errror () {
```

Comment:
This is misspelled because josh chose too, there are lots of this kind of thing. all over the cli. i'd rather tackle massive name changes in the next refactor.